lib/validation-stream.js

```diff
@@ -18,11 +24,17 @@ function ValidationStream(options) {
   // Create internal placeholder for CSS
   this._css = [];
 
-  // Generate our request
-  this.generateRequest();
+  if (isBlank(this.options.text)) {
+    // The W3C service only understands nonempty CSS. Fork a dummy event to be acted on after the
+    // client has yielded.
+    setTimeout(this._emitEmptyCssEvents.bind(this), 0);
```

Comment:
Great thinking here! In Node.js, we can use something even faster than `setTimeout` yet async -- `process.nextTick`.

Also, I always like to document this async action with a comment about preventing zalgo for future devs